app/views/support_interface/application_forms/show.html.erb

```diff
@@ -62,25 +62,33 @@
   <% end %>
 <% end %>
 
-<h2 class="govuk-heading-l govuk-!-margin-top-8" id="qualifications">Qualifications</h2>
 
-<%= render QualificationsComponent.new(application_form: @application_form) %>
-
-<h2 class="govuk-heading-l govuk-!-margin-top-8">Work history</h2>
-
-<%= render WorkHistoryReviewComponent.new(application_form: @application_form, editable: false, heading_level: 3) %>
-
-<h2 class="govuk-heading-l govuk-!-margin-top-8">Unpaid experience and volunteering</h2>
-
-<%= render VolunteeringReviewComponent.new(application_form: @application_form, editable: false, heading_level: 3) %>
-
-<h2 class="govuk-heading-l govuk-!-margin-top-8">Language skills</h2>
-
-<%= render LanguageSkillsComponent.new(application_form: @application_form) %>
-
-<h2 class="govuk-heading-l govuk-!-margin-top-8">Personal statement</h2>
-
-<%= render PersonalStatementComponent.new(application_form: @application_form) %>
+<h2 class="govuk-heading-l govuk-!-margin-top-8">Application data</h2>
+
+<div class="govuk-tabs" data-module="govuk-tabs">
+  <h2 class="govuk-tabs__title">
+    Contents
+  </h2>
+  <ul class="govuk-tabs__list">
+    <li class="govuk-tabs__list-item">
```

Comment:
```suggestion
    <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
```

Reply:
Seems like one of the panels should be selected by default, according to http://govuk-frontend-review.herokuapp.com/components/tabs